includes/loader.php

```diff
@@ -14,6 +14,13 @@
 define( 'GOOGLESITEKIT_PLUGIN_BASENAME', plugin_basename( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
 define( 'GOOGLESITEKIT_PLUGIN_DIR_PATH', plugin_dir_path( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
 
+// Enable this ENV variable to load the new, refactored JavaScript code.
+if ( false !== getenv( 'GOOGLESITEKIT_JS_REFACTOR' ) ) {
+	define( 'GOOGLESITEKIT_JS_REFACTOR', bool( getenv( 'GOOGLESITEKIT_JS_REFACTOR' ) ) );
+} else {
+	define( 'GOOGLESITEKIT_JS_REFACTOR', false );
+}
+
```

Comment:
See above, I think these lines should be removed.

Reply:
These were removed in the other PR https://github.com/google/site-kit-wp/pull/934/commits/35d58188f3815de5fbb46cddb01a20ceb9d99a3e but that hasn't been mirrored here yet.